File: java/1006.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        double a, b, c, average;
        final int PA = 2, PB = 3, PC = 5;
        Scanner input = new Scanner(System.in);
        
        a = input.nextDouble();
        b = input.nextDouble();
        c = input.nextDouble();
        
        average = ((a * PA) + (b * PB) + (c * PC)) / (PA + PB + PC);
        
        System.out.printf("MEDIA = %.1f\n", average);
    }
}

File: src/c/1006.c
#include <stdio.h>
 
int main() {
    double a, b, c, average;
    const int PA = 2, PB = 3, PC = 5;
    
    scanf("%lf", &a);
    scanf("%lf", &b);
    scanf("%lf", &c);
    
    average = ((a * PA) + (b * PB) + (c * PC)) / (PA + PB + PC);
    
    printf("MEDIA = %.1lf\n", average);
 
    return 0;
}
